app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -45,10 +40,15 @@ class MainActivity : AppCompatActivity() {
             val city = response.name
             val humidity = response.main.humidity.toString() + "%"
             val pressure = response.main.pressure
-            val tempRange = round((response.main.temp_min -273)*9/5 + 32).toString() + " to " + round((response.main.temp_max -273)*9/5 + 32).toString() + " Degrees"
-            Toast.makeText(this, tempRange, Toast.LENGTH_LONG).show()
+
+            //Formats temperature
+            fun formatText(temp:Float):String{
+                return round((temp -273)*9/5 + 32).toInt().toString()
```

Comment:
I converted temperatures to integers before converting the temperature to a string because a temperature is never listed as a decimal in any publication.

Reply:
It looks like they are already using round()... so int truncation may be redundant.